python/cudaq/__init__.py

```diff
@@ -129,6 +129,14 @@ def __clearKernelRegistries():
 
 initKwargs = {}
 
+# Look for --target=<target> options
```

Comment:
Note: I'd like to move away from using both `-` and `--` as option prefixes in `nvq++`. The proposal in #1265 is to use a `-` in most cases.

Reply:
Ah, yes.... I thought I remembered some activity or discussion about something related to this recently, and that issue was probably it. Yes, I imagine we should update this code (and the code right below it) as part of any future PR that is posted for that issue. I'll add a comment to that issue so we don't forget.